components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/java/org/wso2/carbon/idp/mgt/ui/util/IdPManagementUIUtil.java

```diff
@@ -559,58 +562,72 @@ private static void buildSCIMProvisioningConfiguration(IdentityProvider fedIdp,
             fedIdp.setDefaultProvisioningConnectorConfig(proConnector);
         }
 
+        if (paramMap.get("scim-version") != null) {
```

Comment:
shall we use the constant SCIM_PROTOCOL_VERSION here and all the other places where "scim-version" is used?